index.js

```diff
@@ -24,22 +24,36 @@ class AddLambdaInsights {
     this.hooks = {
       'before:package:setupProviderConfiguration': this.addLambdaInsights.bind(this),
     };
+    if (this.checkValidationSupport(serverless)) {
+      serverless.configSchemaHandler.defineFunctionProperties('aws', {
```

Comment:
Can you throw an error if not valid version, with a message indicating that the plugin is not compatible with the serverless version used?

Reply:
as discussed offline - we keep it without throwing an error - to allow usage for older serverless versions.